tests/gui/components/update-notifier.spec.js

```diff
@@ -16,19 +17,17 @@ describe('Browser: UpdateNotifier', function() {
     describe('.shouldCheckForUpdates()', function() {
 
       let UpdateNotifierService;
-      let SettingsModel;
       let UPDATE_NOTIFIER_SLEEP_DAYS;
 
-      beforeEach(angular.mock.inject(function(_UpdateNotifierService_, _SettingsModel_, _UPDATE_NOTIFIER_SLEEP_DAYS_) {
+      beforeEach(angular.mock.inject(function(_UpdateNotifierService_, _UPDATE_NOTIFIER_SLEEP_DAYS_) {
         UpdateNotifierService = _UpdateNotifierService_;
-        SettingsModel = _SettingsModel_;
         UPDATE_NOTIFIER_SLEEP_DAYS = _UPDATE_NOTIFIER_SLEEP_DAYS_;
       }));
 
       describe('given the `sleepUpdateCheck` is disabled', function() {
 
         beforeEach(function() {
-          SettingsModel.set('sleepUpdateCheck', false);
+          settings.set('sleepUpdateCheck', false);
```

Comment:
Good question, unless the unit tests mock localStorage it would modify the actual Etcher settings I believe. I'm unsure myself, any suggestions on a way forward here?